src/game/logic/object/update/aiupdate.h

```diff
@@ -17,15 +17,169 @@
 #include "object.h"
 #include "updatemodule.h"
 class INI;
+class AICommandParms;
+class DozerAIInterface;
+class SupplyTruckAIInterface;
+class WorkerAIInterface;
+class HackInternetAIInterface;
+class AssaultTransportAIInterface;
+class JetAIUpdate;
+class AIStateMachine;
+class Locomotor;
+
+enum GuardTargetType
+{
+    TARGET_UNK
+};
+enum AIFreeToExitType
+{
+    FREE_TO_EXIT_UNK
+};
+enum GuardMode

```

Comment:
`GuardModeType` maybe to be consistent with the other enum names here?

Reply:
GuardMode is the official name.